### ISSUE TITLE
Add passportjs middleware for validating authen user to protect /secure route

### ISSUE THREAD

**Issue by junyada100** [VaultExpress/vault-express#86]
Use simple function as passport.js example

**Comment by pradhyumn94** [comment]
@junyada100  If it is still open I would love to work on it.
I am beginner though not in node but in open-source contribution.

**Comment by junyada100** [comment]
yes, this is still an open issue, you may work on it, let me know if you are stuck on anything

**Comment by junyada100** [comment]
Hey @pradhyumn94, currently, i got a problem with passportjs deserialize user info which might/should impact to this issue. you may hold this work a bit until i solve that, will let you know when i done.

**Comment by junyada100** [comment]
@pradhyumn94, you can work further on this issue, the problem was solved.
